meetings/2019-05-14.md

```diff
@@ -0,0 +1,16 @@
+# TSC Meeting May 14, 2019 @12PM MST
+
+## [Zoom Link](https://zoom.us/j/346461536)
```

Comment:
Can you update to https://zoom.us/j/814763419 or we can merge and I'll fix with a new PR?